docs/api/classifier/classification_report.rst

```diff
@@ -5,42 +5,31 @@ Classification Report
 
 The classification report visualizer displays the precision, recall, F1, and support scores for the model. In order to support easier interpretation and problem detection, the report integrates numerical scores with a color-coded heatmap. All heatmaps are in the range ``(0.0, 1.0)`` to facilitate easy comparison of classification models across different classification reports.
 
-.. code:: python
+.. plot::
+    :context: close-figs
```

Comment:
per my conversation with Ben.. @bbengfort is it ok to use it here?

Reply:
Thanks for the feedback...For the future, I will reference other conversations better. so that you don't have to dig around.

Reply:
No worries! Just wanted to make sure I didn't miss something :)